### DISCUSSION TITLE
Lost admin access

### DISCUSSION THREAD

**Discussion by tobiasredel** [seerr-team/seerr#950] [General]
I replaced my jellyseerr server with a new instance, now my user somehow is not admin anymore but just normal user - any way to set my user as admin from the command line ?

**Comment by fallenbagel** [comment]
A change in server = to change is userids, serverids, etc.

You'll need to edit the db file and change the jellyfinUserId field of ID=1 to your current jellyfinUserId

**Reply by tobiasredel** [reply]
Thank you - that did the trick

**Reply by Dirty504** [reply]
I did the same thing.  What's the path to this db file please?

**Reply by fallenbagel** [reply]
> I did the same thing.  What's the path to this db file please?

In your config folder

**Comment by BourbonDoc** [comment]
I'm having the same issue but it's not a new install. I install the instance, only have the single user I setup with, everything works great. Come back the next day and all the admin options are gone. This has happened with 2 tries of fresh installs. I'm running it on TrueNAS Scale, and am having difficulty finding a db file to edit. I located settings.json and didn't see user options. In the db directory are 3 "dbsqlite3" files that seem to be mostly non human readable with blocks about the media. Nothing seems to have a user ID field. Help is appreciated!

**Comment by fallenbagel** [comment]
> I'm having the same issue but it's not a new install. I install the instance, only have the single user I setup with, everything works great. Come back the next day and all the admin options are gone. This has happened with 2 tries of fresh installs. I'm running it on TrueNAS Scale, and am having difficulty finding a db file to edit. I located settings.json and didn't see user options. In the db directory are 3 "dbsqlite3" files that seem to be mostly non human readable with blocks about the media. Nothing seems to have a user ID field. Help is appreciated!

We carry out support on discord. Please hop onto discord.
